doc/api/indexes.rst

```diff
@@ -0,0 +1,34 @@
+.. currentmodule:: xarray
+
+Indexes
+=======
+
+Default, pandas-backed indexes built-in to Xarray:
+
+.. autosummary::
+   :toctree: ../generated/
+
+   indexes.PandasIndex
+   indexes.PandasMultiIndex
+
+
+More complex indexes built-in to Xarray:
+
+.. autosummary::
+   :toctree: ../generated/
+
+   CFTimeIndex
+   indexes.RangeIndex
+   indexes.NDPointIndex
```

Comment:
Why does the Indexes dropdown (left-hand sidebar only list these 3) but not PandasIndex?... I feel like I'm going crazy and missing something obvious, but why do some left-hand sidebar sub-pages have dropdowns (DataArray, Indexes) and others don't (Dataset, Coordinates)?